### PR TITLE
Fix Issues with Effects Tablist Widget

### PR DESCRIPTION
**Pull request by NopoTheGamer**
- Fixes booster cookie warning to use widget
- Fixes god pot on todo not updating with new widget
- adds the footer to the tablist lines

### CODE REVIEW

**Comment by NopoTheGamer** [comment]
Fixes https://github.com/NotEnoughUpdates/NotEnoughUpdates/issues/1063

File: src/main/java/io/github/moulberry/notenoughupdates/util/TabListUtils.java [hunk 1]
```diff
@@ -87,6 +88,33 @@ private List<String> getTabList0() {
 			String name = Minecraft.getMinecraft().ingameGUI.getTabList().getPlayerName(info);
 			result.add(name);
 		}
+
+		ArrayList<String> lines = null;
+		try {
+			String[] footer = ((AccessorGuiPlayerTabOverlay) Minecraft.getMinecraft().ingameGUI.getTabList())
+				.getFooter().getFormattedText()
+				.split("\n");
+			//System.out.println();
```
Comment (jani270) [inline src/main/java/io/github/moulberry/notenoughupdates/util/TabListUtils.java:97]:
```suggestion
```

**Comment by jani270** [comment]
Worked for me cant say much about the code tho

File: src/main/java/io/github/moulberry/notenoughupdates/overlays/TimersOverlay.java [hunk 2]
```diff
@@ -517,9 +502,35 @@ public void update() {
...
+						case "Minute":
+						case "m":
+							godPotDuration += godpotRemainingTime * 60 * 1000;
+							break;
+						case "Seconds":
+						case "Second":
+						case "s":
+							godPotDuration += godpotRemainingTime * 1000;
+							break;
+					}
```
Comment (nea89o) [inline src/main/java/io/github/moulberry/notenoughupdates/overlays/TimersOverlay.java:532]: We already have another method (getMinutesRemaining) doing practically the same. I think we should join them.

**Review by nea89o — CHANGES_REQUESTED** [review]

**Review by Lulonaut — APPROVED** [review]

**Review by nea89o — APPROVED** [review]

**Pull request merged**